spec/models/miq_server/environment_manager_spec.rb

```diff
@@ -3,14 +3,14 @@
 RSpec.describe "Server Environment Management" do
   context ".get_network_information" do
     let(:mac_address) { 'a:1:b:2:c:3:d:4' }
-    let(:hostname)    { "localhost.localdomain" }
+    let(:hostname)    { "localhost" }
```

Comment:
This is interesting, when I run the old method `Socket.gethostbyname(Socket.gethostname).first` with `localhost` I don't get `localhost.localdomain` back:
```
(irb):2: warning: Socket.gethostbyname is deprecated; use Addrinfo.getaddrinfo instead.
=> "localhost"
irb(main):003:0> Socket.gethostbyname('localhost.localdomain').first
(irb):3: warning: Socket.gethostbyname is deprecated; use Addrinfo.getaddrinfo instead.
=> "localhost"
```
So I think this test was misleading.  I tried to get `Addrinfo.tcp("localhost.localdomain", nil)` to yield `localhost.localdomain` but it just doesn't.